build.sbt

```diff
@@ -279,6 +280,18 @@ lazy val sse = project
   .settings(allSettings)
   .dependsOn(core)
 
+lazy val refined = project
+  .settings(moduleName := "finch-refined")
+  .settings(allSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "eu.timepit" %% "refined" % refinedVersion,
+      "eu.timepit" %% "refined-cats" % refinedVersion,
```

Comment:
It looks like refined-cats could be a test dependency, too.

Reply:
Good catch, thanks